### PR TITLE
add the code in try and catch block

### PR DESCRIPTION
**Pull request by Vayras**
add the code in try and catch block this is done so to handle when the response is a non axios error

### CODE REVIEW

**Comment by arthurfiorette** [comment]
Hey @Vayras, thanks for your contribution.

I should've written a more descriptive issue, but what I was saying was just to detect in the first reject response interceptor, if the error was an axios error. (which can be checked with `!!error.isAxiosError`)

File: src/interceptors/response.ts [hunk 1]
```diff
@@ -236,124 +236,131 @@ export function defaultResponseInterceptor(
...
-    ) {
-      await rejectResponse(id, config);
-
-      if (__ACI_DEV__) {
-        axios.debug?.({
-          id,
-          msg: 'Caught an error in the request interceptor',
-          data: { cache, error, config }
-        });
+    try {
```
Comment (arthurfiorette) [inline src/interceptors/response.ts:239]: reset everything else :)

**Comment by Vayras** [comment]
@arthurfiorette thank you for letting me know . Let me know if I can contribute more and help

**Comment by arthurfiorette** [comment]
I've made a review with some changes and a test case would be awesome :)

**Comment by arthurfiorette** [comment]
Implemented by 929054e

**Pull request closed without merging**